Assets/Scripts/PlayerController/PlayerMovement.cs

```diff
@@ -27,6 +27,9 @@ public class PlayerMovement : MonoBehaviour
     [SerializeField]
     private float m_JumpHeight = 1.3f;
 
+    [SerializeField]
+    private GameObject bomb; 
+
```

Comment:
Should `PlayerMovement` really contain a `Bomb` game object out of nowhere?

Reply:
Truth! I think will create a separate script for that after done with the powerups!